mux/spec/cli.md

```
@@ -61,6 +63,11 @@ The generated CLI requires one of `--index` or `--delta` for `select-tab`, `sele
 | `new-screen` | implemented | none | `--workspace <id>`, `--cols <n> --rows <n>` | surface id |
 | `split` | implemented | `--pane <id> --dir right|down` | `--cols <n> --rows <n>` | surface id |
 | `set-ratio` | implemented | `--pane <id> --dir right|down --ratio <n>` | none | none |
+| `pane-neighbor` | implemented | `--pane <id> --dir left|right|up|down` | none | pane id or `null` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)